package.json

```diff
@@ -82,6 +87,7 @@
     "react-hook-form": "~5.7.2",
     "react-select": "~3.1.0",
     "shp-write": "~0.3.2",
+    "ugh": "^0.1.1",
```

Comment:
yooo ugh is an internal util (see `~/src/ugh.js`)

Reply:
kk removing

Reply:
@AtherBilal hey this one is still in there